libs/node_resolver/analyze.rs

```diff
@@ -387,7 +387,7 @@ impl<
               &package_subpath,
               exports,
               Some(&referrer),
-              ResolutionMode::Import,
+              ResolutionMode::Require,
```

Comment:
I'm just going to test with what Node does before undrafting to ensure I'm not actually breaking stuff.